Fabric.Terminology.SqlServer/Models/EmptySamdBinding.cs

```diff
@@ -0,0 +1,23 @@
+﻿// ReSharper disable InconsistentNaming
+
+namespace Fabric.Terminology.SqlServer.Models
+{
+    using System;
+
+    /// <summary>
+    /// Provides HC specific binding information in instances where field information is required
+    /// but no binding was used to create the record.
+    /// </summary>
+    internal class EmptySamdBinding
+    {
+        public int BindingID => -1000;
+
+        public string BindingNM => "ValueSetCodeClient";
+
+
+        public DateTime GetLastLoadDTS(DateTime? lastLoad = null)
```

Comment:
This relates to **Create a list of irrelevant ClientTerm fields** #21.  Will change to either default(DateTime) or DateTime.UtcNow if LastLoadDT field remains.